### ISSUE TITLE
Barbarian rage not applying on path of Wild Magic.

### ISSUE THREAD

**Issue by tehguitarist** [DFreds/dfreds-convenient-effects#143]
Not sure if this is related to the recent dnd5e system update, but it seems that the rage effect is no longer working. Error message from console below. If I right click and hit "toggle status effect" it throws up an error stating I need to reload foundry. Happy to get more details if needed! When I  tried applying the effect to a fighter, it threw an error about it not being a barbarian, but did work. If I delete the subclass from the actor, it also applies again, so I suspect it's either how the new system is handling subclasses, or it's that subclass in particular. Character imported with DDB-Importer

```
Uncaught (in promise) TypeError: barbarianClass.data.data.subclass.toLowerCase is not a function
[Detected 2 packages: dfreds-convenient-effects, socketlib]
    _addResistancesIfTotemWarrior https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/effects/dynamic-effects-adder.js:140
    _addRageEffects https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/effects/dynamic-effects-adder.js:118
    addDynamicEffects https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/effects/dynamic-effects-adder.js:22
    addEffect https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/effects/effect-handler.js:101
    toggleEffect https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/effects/effect-handler.js:30
    _executeLocal https://192.168.0.23:30000/modules/socketlib/src/socketlib.js:192
    executeAsGM https://192.168.0.23:30000/modules/socketlib/src/socketlib.js:90
    toggleEffect https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/effect-interface.js:114
    onEffectClick https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/app/convenient-effects-controller.js:246
    jQuery 8
    _initClickListeners https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/app/convenient-effects-app.js:107
    activateListeners https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/app/convenient-effects-app.js:42
    _render https://192.168.0.23:30000/scripts/foundry.js:3090
    render https://192.168.0.23:30000/scripts/foundry.js:3021
    _handleConvenientEffectsClick https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/controls.js:40
    _onClickTool https://192.168.0.23:30000/scripts/foundry.js:51831
    jQuery 2
        dispatch
        handle
[dynamic-effects-adder.js:140:53](https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/effects/dynamic-effects-adder.js)
    onEffectClick https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/app/convenient-effects-controller.js:246
    AsyncFunctionThrow self-hosted:642
    (Async: async)
    onEffectClick self-hosted:1121
    jQuery 8
    _initClickListeners https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/app/convenient-effects-app.js:107
    activateListeners https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/app/convenient-effects-app.js:42
    _render https://192.168.0.23:30000/scripts/foundry.js:3090
    AsyncFunctionNext self-hosted:638
    (Async: async)
    render https://192.168.0.23:30000/scripts/foundry.js:3021
    _handleConvenientEffectsClick https://192.168.0.23:30000/modules/dfreds-convenient-effects/scripts/controls.js:40
    _onClickTool https://192.168.0.23:30000/scripts/foundry.js:51831
    _onClickTool self-hosted:1121
    jQuery 2

```

**Comment by DFreds** [comment]
Okay, thanks for reporting this. I'll fix it in the next release

**Comment by DFreds** [comment]
Fixed in v2.6.1
